SWLOR.Game.Server/Service/PlayerStatService.cs

```diff
@@ -473,21 +469,25 @@ public static EffectiveItemStats GetPlayerItemEffectiveStats(NWPlayer player, NW
 
 
                 // Calculate AC
-                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType) ||
-                    ItemService.ShieldBaseItemTypes.Contains(item.BaseItemType))
+                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType))
                 {
                     int skillRankToUse;
+                    int maxAC = 0;
+
                     if (item.CustomItemType == CustomItemType.HeavyArmor)
                     {
                         skillRankToUse = heavyRank;
+                        maxAC = 12;
```

Comment:
Can I get a run down of how you arrived at these caps for each of the armor types?

Reply:
We had a whole big discussion about the caps, Mith Naia and I all thing 12 is a good start, once we get pushed to test we can see how it changes up stuff though.
The idea is that You can only get +20 Attack Bonus, so we wanted to cap AC a little higher then AB. With 12 per item it gets about 4 more AC then AB or 8 in light armor's case. which should put players back into range for PvP content, hopefully.

Reply:
I'm good with this.